Jenkinsfile.nightly

```diff
@@ -18,18 +18,12 @@ pipeline {
 			steps {
 				sh '''
 				sudo service postgresql restart
-				cp test/data/config.json test/data/genesis_block.json .
 				'''
 			}
 		}
-		stage('Run integration tests without syncing') {
+		stage('Run integration tests with syncing only') {
```

Comment:
Why we removed one stage?

Reply:
Because it was brittle and not giving much value. We should create a bigger strategy and pick the correct scenarios to be tested in a small network with a maximum of 10 peers. This review aims to make that possible.

Reply:
We are talking about the removed stage/task about `with broadcasting only`. `with syncing only` is ok. The variable names were a complete mess.